cddl/examples/payload-corim-4.diag

```diff
@@ -0,0 +1,46 @@
+501(/ corim-map / {
```

Comment:
send the corim-map to L2

Also, `ts=2 et`

Reply:
I don't understand your second comment. ts=2?

Reply:
see https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/337/files#r1925086997